src/theme/material/tab-container.m.css

```diff
@@ -33,20 +33,13 @@
 	composes: nativeResetRoot from './button.m.css';
 	position: absolute;
 	top: 50%;
-	transform: translateY(-50%);
+	transform: translateY(-50%) scale(0.7);
```

Comment:
could a `size` property be specified on the `Icon` instead perhaps?

Reply:
I considered that, but the Dojo theme doesn't need it. For whatever reason, the Material Icon is just a little big.